<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fmp</class>
 <widget class="QDialog" name="Fmp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de Matiere Premiere</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QLineEdit" name="IdentProd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant MP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant Produit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_quitter">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>150</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="identMP">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="but_ajouter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_actes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Actes</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_type">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Production</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Conception</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cout</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cout_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableMP">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="but_refresh">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualiser</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>but_quitter</sender>
   <signal>clicked()</signal>
   <receiver>Fmp</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>766</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
